x-pack/legacy/plugins/monitoring/public/components/elasticsearch/nodes/nodes.js

```diff
@@ -110,21 +107,19 @@ const getColumns = (showCgroupMetricsElasticsearch, setupMode, clusterUuid) => {
     field: 'isOnline',
     sortable: true,
     render: value => {
-      const status = value ? i18n.translate('xpack.monitoring.elasticsearch.nodes.statusColumn.onlineLabel', {
-        defaultMessage: 'Online',
-      }) : i18n.translate('xpack.monitoring.elasticsearch.nodes.statusColumn.offlineLabel', {
-        defaultMessage: 'Offline',
-      });
+      const status = value
+        ? i18n.translate('xpack.monitoring.elasticsearch.nodes.statusColumn.onlineLabel', {
+            defaultMessage: 'Online',
+          })
+        : i18n.translate('xpack.monitoring.elasticsearch.nodes.statusColumn.offlineLabel', {
+            defaultMessage: 'Offline',
+          });
```

Comment:
These lines have ESLint errors causing CI to fail

<img width="763" alt="Screen Shot 2019-11-27 at 12 45 20 PM" src="https://user-images.githubusercontent.com/549577/69747211-ee670080-1113-11ea-8965-774855a66f9f.png">